### PR TITLE
add required props for dataexport

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

File: tools/dataMigration/migrateMetadata/getRow.ts [hunk 1]
```diff
@@ -20,5 +20,19 @@ export const getRow = (props: { cycles: Array<string>; rowSpec: RowSpec; table:
...
+    row.props.labelKey = rowSpec.labelKey
+  }
+  if (rowSpec.labelPrefixKey) {
+    row.props.labelPrefixKey = rowSpec.labelPrefixKey
+  }
+  if (rowSpec.labelParams) {
+    row.props.labelParams = rowSpec.labelParams
+  }
+  if (rowSpec.variableExport) {
+    row.props.variableExport = rowSpec.variableExport
```
Comment (minotogna) [inline tools/dataMigration/migrateMetadata/getRow.ts:34]: Now we should only use variableName
All rows have it now

**Review by minotogna — CHANGES_REQUESTED** [review]

File: webapp/sectionSpec/rowSpec.ts [hunk 2]
```diff
@@ -15,9 +15,9 @@ export interface RowSpec {
   calculateFn?: CalculateValue
   chartProps?: RowChartSpec
   idx?: string | number
-  // labelKey?: string
-  // labelPrefixKey?: string
-  // labelParams?: Record<string, string>
+  labelKey?: string
```
Comment (minotogna) [inline webapp/sectionSpec/rowSpec.ts:18]: Let's add now an object label with properties key, params, props
Btw where is it needed ?

File: meta/assessment/row.ts [hunk 3]
```diff
@@ -25,6 +31,8 @@ export interface RowProps {
     labelKey: string
     color: string
   }
+  label?: RowLabel
+  variableExport?: string
```
Comment (sorja) [inline meta/assessment/row.ts:35]: comment/remove variableExport

File: meta/assessment/row.ts [hunk 4]
```diff
@@ -15,6 +15,12 @@ export enum RowType {
   placeholder = 'placeholder',
 }
 
+export interface RowLabel {
+  labelKey?: string
+  labelPrefixKey?: string
+  labelParams?: Record<string, string>
```
Comment (minotogna) [inline meta/assessment/row.ts:21]: call this key, prefix, params

**Review by minotogna — CHANGES_REQUESTED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**